lib/system/excpt.nim

```diff
@@ -474,12 +474,34 @@ when defined(endb):
 when defined(cpp) and appType != "lib" and
     not defined(js) and not defined(nimscript) and
     hostOS != "standalone" and not defined(noCppExceptions):
+  proc verboseTerminate() {.importc: "__gnu_cxx::__verbose_terminate_handler", header: "<exception>".}
```

Comment:
I don't think the stdlib should depends on libstdc++, given that there are macosx and windows targets.

Reply:
`verboseTerminate()` is here only to show what is possible. `std::get_terminate()` would be better, and it's standard C++11. We could test a macro for C++11. Delegating to the normal terminate_handler is definitely what the C++ coder would expect.

The question is what to do for C++ < 11. I'm fine with `quit 1` in that case. I'm also fine with re-throwing, catching, and printing `what()` instead.